### PR TITLE
Update pycurl URL and digest hashes

### PR DESCRIPTION
**Pull request by GldRush98**
_Motivation:_ Pycurl's current version is no longer hosted at the url. The new url has been added.
_Linked issues:_ SynoCommunity/spksrc#2569

### Checklist
- [] Build rule `all-supported` completed successfully
- [ok] Package upgrade completed successfully
- [ok] New installation of package completed successfully

### CODE REVIEW

**Comment by Dr-Bean** [comment]
The first commit (https://github.com/SynoCommunity/spksrc/pull/2570/commits/4c9e86d06ad93cdb343d72f92bf320f8ccbbc62f) doesn't belong in the PR. We might be better off cherry-picking the second commit only. Maybe you can remove it (start with a fresh fork, make the changes, push to remote for example)
Other than that, content LGTM, thanks :)

**Comment by GldRush98** [comment]
Yeah, I had a feeling I didn't do that right when I saw my initial sync commit int here.  Let me try again.  Sorry, I'm kind of a newb at this stuff.
